classes/class-wc-connect-shipping-label.php

```diff
@@ -39,7 +39,7 @@ protected function get_items_as_individual_packages( $order ) {
 				$name = html_entity_decode( $product->get_formatted_name() );
 
 				for ( $i = 0; $i < $item[ 'qty' ]; $i++ ) {
-					$packages[] = array(
+					$packages[ "weight_{$i}_individual" ] = array(
```

Comment:
You can do either, I'm not sure one way is more correct than the other.

See "Complex (curly) syntax" example here: https://secure.php.net/manual/en/language.types.string.php#language.types.string.syntax.double